lib/DDG/Meta/Data.pm

```diff
@@ -181,6 +185,15 @@ sub get_js {
 # return a hash of IA objects by id
 sub by_id { $ia_metadata{id} }
 
+# return a hash of IA metadata for fatheads
+sub by_source { $ia_metadata{source} }
```

Comment:
Let's make these "by_fathead_*" so people don't see them being used, assume they work for all IAs, and wonder why they only get fathead data.